README.md

```diff
@@ -171,27 +183,27 @@ All these three subcommands accept the following options:
 --dry-run: Dry run execution
 ```
 
-#### Anatomy of the configuration file
+### Anatomy of the configuration file
 
 The self-provisioning feature uses a different configuration file than the `Furyfile.yml`.
 Use the `Furyfile.yml` file while using package-manager features.
```

Comment:
ok, but what is the file that uses?
I'd change the last sentence from:
"Use the `Furyfile.yml` file while using package-manager features."
to
"While the `Furyfile.yml` file is used by the package-manager features, the self-provision features use a separated `cluster.yml` file:"